ios/FluentUI.Demo/FluentUI.Demo/Demos/NavigationControllerDemoController.swift

```diff
@@ -173,6 +178,45 @@ class RootViewController: UIViewController, UITableViewDataSource, UITableViewDe
         return tableView
     }()
 
+    private(set) lazy var searchProgressSpinnerSwitchView: UIView = {
+        let itemRow = UIStackView()
+        itemRow.axis = .horizontal
+        itemRow.distribution = .equalCentering
+        itemRow.alignment = .leading
+        itemRow.isLayoutMarginsRelativeArrangement = true
+        itemRow.layoutMargins = UIEdgeInsets(top: 10, left: 15, bottom: 10, right: 15)
+        itemRow.translatesAutoresizingMaskIntoConstraints = false
+
+        let searchSpinnerSwitchLabel = Label(style: .subhead, colorStyle: .regular)
+        searchSpinnerSwitchLabel.text = "Show spinner while using the search bar:"
```

Comment:
super nit: I think we don't need the ":" at the end of the string

Reply:
removed

---
In reply to: [470811273](https://github.com/microsoft/fluentui-apple/pull/174#discussion_r470811273) [](ancestors = 470811273)

Reply:
Thank you! :)